### DISCUSSION TITLE
Failed to download vulnerability DB

### DISCUSSION THREAD

**Discussion by sbernard31** [aquasecurity/trivy#7783] [Q&A]
I recently move from trivy 0.48.2 to 0.56.2 and on jenkins my job failred with : 

```
FATAL	Fatal error	init error
   :DB error: failed to download vulnerability DB
   :OCI artifact error: failed to download vulnerability DB: failed to download artifact from any source
```

Do you have any idea what could cause that ? 
And also I see that even if I get that error trivy seems to be  able to detect some vulnerabilities ? How that is possible ?

See : https://ci.eclipse.org/leshan/job/leshan-weekly/job/1.x/46/console

**Comment by simar7** [comment]
Please see https://github.com/aquasecurity/trivy/discussions/7699
